3ds2/src/main/java/com/adyen/checkout/adyen3ds2/Adyen3DS2ComponentProvider.kt

```diff
@@ -35,17 +36,19 @@ class Adyen3DS2ComponentProvider :
     override fun <T> get(
         owner: T,
         application: Application,
-        configuration: Adyen3DS2Configuration
+        configuration: Adyen3DS2Configuration,
+        key: String?,
```

Comment:
Do we need `key: String? = null` here as well or no?

Reply:
It's not possible because you can only set default value in the interface